desktop/src/main/java/bisq/desktop/main/offer/MutableOfferDataModel.java

```diff
@@ -282,6 +282,8 @@ public boolean initWithData(OfferPayload.Direction direction, TradeCurrency trad
         setTradeCurrencyFromPaymentAccount(paymentAccount);
         tradeCurrencyCode.set(this.tradeCurrency.getCode());
 
+        setTradeCurrencyFromPaymentAccount(paymentAccount);
```

Comment:
I don't understand why this needs to be done twice, it's set three lines up.

Reply:
Good catch was added by mistake.